sale_order_line_price_history/data/price_history_parameters.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo noupdate="1">
+    <record
+        id="sale_order_line_price_history_order_line_limit"
+        model="ir.config_parameter"
```

Comment:
**suggestion:** It is better if you don't include these kind of records in data. Instead, when you search for the parameter, assign a default value.

The reason is that some admin could go to system parameters and remove the parameter, and re-create it later.

In these cases, when later you upgrade the module, it will try to re-create the parameter (because the parameter exists by key, but its XMLID has been removed).